packages/material-ui-system/src/useTheme.js

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { useTheme as muiUseTheme } from '@material-ui/private-theming';
+import { ThemeContext } from '@material-ui/styled-engine';
+import createTheme from './createTheme';
+
+function isObjectEmpty(obj) {
+  return Object.keys(obj).length === 0;
+}
+
+export const systemDefaultTheme = createTheme();
+
+function useTheme(defaultTheme = systemDefaultTheme) {
```

Comment:
I added the `defaultTheme` as a first arg of the hook. The `useTheme` from the system uses the system default theme, but we need the Material-UI theme in the core, that's why it is added.